SparkPost/2web2ui
# SA-966 Display test results grouped by region

### What Changed
 - Added Select filter in "Placement Breakdown" section with 2 options: Provider and Region. (/inbox-placement/details/73/details).
 - Upon selection of "Region" option, table data below will change to be grouped by region instead of provider.

### How To Test
 - Open inbox placement page and ensure the list has data with the "Provider" option being the default. 
 - Change the selection to "Region" and ensure the data in the table lists the names of each region in the test in start case form. (i.e. North America as opposed to north america).
 - Change back to "Provider" option and make sure the data changes back.
